src/components/Scheme/ReputationFromToken.tsx

```diff
@@ -333,7 +337,7 @@ class ReputationFromToken extends React.Component<IProps, IState> {
                 </div>
                 <div className={schemeCss.redemptionButton}>
                   <button type="submit"
-                    disabled={false}
+                    disabled={this.state.alreadyRedeemed || this.state.redemptionAmount.isZero()}
```

Comment:
It is likely that just the css will not prevent the button from being clicked.  You may also need to set the `onClick` to `undefined`.

Reply:
this is not css this is the button disabled property and it will prevent the button from being clicked

Reply:
OK, I think I need to not review code when I'm not awake yet.